
%\pdfoutput 1 %pdf

\documentclass[11pt]{article}

\usepackage{amssymb,amsmath,amsthm}
\usepackage{verbatim}
\usepackage{fullpage}
\usepackage{gencor}
\frenchspacing

%\setlength\parindent{0pt}

\title{Estimating Genetic Correlation from GWAS Summary Statistics}
\author{Brendan Bulik-Sullivan*, Hilary Finucane*, ... et al}
\date{July 8, 2014}

\begin{document}
\maketitle



% Main themes
% - Genetic correlation is important because 
% - relationships between phenotypes obv interesting
% - phenotypic correlation confounded by environmental factors 
% - particular promise for looking at intermediate phenotypes \cite{Do et al}

% - summary statistics only --> many more genetic correlation possible
% - immune to sample overlap and pop strat
% 
% - more flexible models of genetic architecture
% - account for MAF and LD dependence
% - make no distributional assumptions (causal SNPs in LD are fine)
% - work for case/control traits
%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{abstract}\label{abstract}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Discovering relationships between phenotypes is a fundamental goal of traditional epidemiology.
Phenotypic correlations in observational epidemiological studies are often confounded by environmental factors,
so genetic correlations between phenotypes may be more easily interpretable.
The largest currently available sources of genotype-phenotype data are genome-wide association studies (GWAS); 
however, existing methods for estimating genetic correlation from GWAS data require
genotype and phenotype data for at least one of the phenotypes, 
which is often impossible to obtain due to restrictions on data sharing.
For this reason, only a few dozen genetic correlations have been estimated from GWAS data to date.
In this paper, we describe a method based on LD Score regression
which estimates genetic correlations directly from GWAS summary statistics
and is immune to sample overlap. 
Since dozens of sets of summary statistics can be freely downloaded from the internet,
we can report a much larger number of  genetic correlations 
 -- more than 700 in this paper alone -- than was previously possible.
In addition, we relax many standard assumptions about genetic architecture, 
and demonstrate that our method will give the right answer
even when effect size depends on allele frequency or linkage disequilibrium.

\end{abstract}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Introduction}
\label{Introduction}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Investigating relationships between the genetic etiologies of heritable phenotypes is important for reasons 1-3.
Existing methods that use GWAS summary statistics do not properly account for LD, can be biased by sample overlap between studies, or only look at significant SNPs.
Since at current sample sizes, genome-wide significant SNPs generally account for at most ~10\% of the heritability explained by common SNPs, methods that rely exclusively on top SNPs are likely to have low power relative to methods that use all common SNPs. 
In addition, restricting to significant SNPs is not viable for phenotypes where there exist large GWAS and no significant SNPs. 
For instance, it may be necessary to genotype hundreds of thousands of cases in order to find a significant locus for a common disease phenotype with low heritability and quasi-infinitesimal genetic architecture (e.g., major depression), yet for such a phenotype, it is often possible to obtain a reliable estimate of genetic correlation with only tens of thousands of samples.
One solution is to use all the SNPs to estimate genetic correlation, defined as the correlation in the population between the additive genetic components of the phenotypes (or for binary phenotypes, the correlation in the population between additive genetic components of liability).

Existing methods for estimating genetic correlation from genotype data (e.g., REML as implemented in the software package GCTA \cite{yang2010, yang2011gcta}) are computationally complex 
and require genotype data, which can be laborious to obtain. 
For this reason, papers typically report at most a handful of genetic correlations, estimated from samples of at most a few tends of thousands of individuals.
We propose a modification of LD Score regression \cite{buliksullivan2014} that can estimate genetic correlation from GWAS summary statistics. 
Specifically, by regressing the product $z_{1,j}z_{2,j}$ of $Z$-scores two GWAS against $\ell_j$, the LD Score of SNP $j$, the slope times a constant estimates genetic covariance.
Since sample overlap affects the term $z_{1,j}z_{2,j}$ equally for all SNPs, this method is not biased by sample overlap.
In addition, we show that this method is robust to many common confounders, including population stratification, cryptic relatedness and misspecified models of genetic architecture.
Since the method is computationally trivial and requires only GWAS summary statistics, many of which are publicly available, we are able report more than 500 genetic correlations.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Results}\label{Results}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\subsection{Simulations}
In order to check our derivations and verify the robustness of our inference procedure to 
violations of our modeling assumptions, we performed a variety of simulations. 

\subsubsection{Sample Overlap}
To verify the robustness of our inference procedure to sample overlap, 
we simulated two GWAS with quantitative phenotypes,
using genotypes from the 4,292 individuals in the WTCCC1 bipolar disorder cohort for the first GWAS
and genotypes from the 4,482 individuals in the WTCCC1 coronary artery disease cohort for the second GWAS.
These cohorts contain 2,713 overlapping individuals. 
Additive genetic effect sizes were drawn from a bivariate point-normal distribution 
(see the Supplementary Note for a definition of this distribution) 
with 10\% of SNPs causal and true genetic correlation $0.7$.
We then estimated genetic correlation using LD Score regression.
Results from these simulations are summarized in [\textbf{TABLE N}],
and confirm that LD Score regression is not biased by sample overlap.

\subsubsection{Case-Control Ascertainment}
Next, we simulated ascertained GWAS in order to evaluate the performance of 
LD Score regression under various case/control ascertainment schemes.
Simulating case/control GWAS under a liability threshold model requires rejection
sampling from a large pool of genotypes.
For instance, in order to simulate drawing 1,000 cases 
for a phenotype with prevalence of $1\%$, 
one would need on expectation to sample 100,000 genotypes.
We do not have access to genotypes for 100,000 individuals, 
so we used simulated genotypes with a simplified LD structure 
($r^2=0$ or $1$) for all simulations with ascertainment.

First, we simulated standard case/control ascertainment following a liability threshold model

We then estimated the genetic correlation 
(specifically, the correlation in the population between genetic liabilities)
using LD Score regression. 
Results from these simulations are summarized in [\textbf{TABLE N}],
and confirm that LD Score regression is effectively unbiased even with heavy ascertainment.

\subsubsection{Complicated Ascertainment}
Next, we simulated a more complicated ascertainment scheme,
representative of the study design used by many large GWAS consortia,
where all case samples are independent, 
but there is a large pool of healthy controls 
(\emph{i.e.,} individuals who are controls for both phenotypes)
shared between all studies.
...
We caution that while LD Score regression estimates of genetic correlation are robust to
standard case/control ascertainment and the healthy controls model of ascertainment, 
it can be difficult to interpret LD Score regression estimates of genetic covariance 
obtained from GWAS with more complicated ascertainment schemes.
As an example, 
if one were to attempt to estimate the genetic correlation between body-mass index (BMI) and type-2 diabetes (T2D)
using LD Score regression and summary statistics from a non-ascertained GWAS for BMI
and a GWAS for T2D consisting of high-BMI controls and low-BMI cases,
then the resulting estimate would not be on a readily-interpretable scale.

\subsubsection{Robustness to Misspecified Models of Genetic Architecture}
Estimates of heritability and genetic covariance can be biased if the underlying model of genetic architecture is misspecified.
For example, (Speed et al) demonstrate that REML can be strongly biased by MAF- or LD-dependent genetic architectures, 
and LD Score regression is subject to similar biases. 
It is possible to remove these biases by employing a more complex regression procedure that simultaneously infers both heritability and genetic architecture, for instance MAF-binned LD Score regression or LD Score-binned LD Score regression, 
but the increase in model complexity comes with a commensurate increase in standard error. 

Since genetic correlation is estimated as a ratio $\hat{\rho}_g / \sqrt{\hat{h}^2_1\hat{h}^2_2}$, model misspecification bias in the numerator and model misspecification bias in the the denominator will tend to approximately cancel. As a result, we expect simpler models tend to perform better as measured by mean square error (MSE).
To test this hypothesis, we simulated a variety of realistic genetic architectures with MAF- and LD-dependence (WHICH MODELS?), and estimated genetic correlation using both naive LD Score regression and a more sophisticated inference procedure (MAF x LD-binned LD Score regression) that accounts for MAF and LD dependence. 
As expected, naive LD Score regression shows no discernible directional bias and gives better MSE for genetic correlation estimation.
Results from these simulations are summarized in supplementary table NNN.
We note that this result holds only for genetic correlation, not for heritability or genetic covariance.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Real Data}\label{Real Data}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsubsection{Replication of PGC Cross Disorder Results}

As a sanity check, we replicated the genetic correlation results obtained with
raw genotypes and GCTA/REML in
the PGC Cross-Disorder Group paper \cite{pgccdg2013}, 
using the summary statistics from \cite{cross2013identification} downloaded from the PGC website (URLs).
For this replication, we used an LD Score with $r^2$'s from the 1000 Genomes Europeans
but with the sum of $r^2$'s taken only over SNPs in HapMap 3 
(hereafter referred to as HapMap3 LD Score)
because this is most similar to the model of genetic architecture implied by GCTA/REML,
where only the effects of genotyped SNPs are modeled
(see the section ``Relationship to Haseman-Elston Regression'' in the Supplementary Note). 
The results from the PGC Cross-Disorder Group paper replicated closely, 
and the standard errors were similar to those obtained from GCTA/REML.

\subsubsection{Application to a Large Set of Publicly Available Summary Statistics}
Finally, we applied our method to a large set of publicly available summary statistics, 
including studies of 
schizophrenia\cite{schizophrenia2014biological}, 
major depression\cite{ripke2012mega}, 
bipolar disorder\cite{sklar2011large}, 
autism\cite{cross2013identification},
ADHD\cite{neale2010meta},
height\cite{allen2010hundreds}, 
body mass index\cite{speliotes2010association}, 
waist-hip ratio\cite{heid2010meta}, 
obesity\cite{berndt2013genome}, 
extreme height\cite{berndt2013genome}, 
various insulin- and glucose- related traits\cite{prokopenko2014central,scott2012large,manning2012genome,strawbridge2011genome,saxena2010genetic,dupuis2010new,soranzo2010common},
coronary artery disease\cite{schunkert2011large}, 
type-2 diabetes\cite{morris2012large}, 
rheumatoid arthritis\cite{stahl2010genome}, 
plasma lipid traits\cite{teslovich2010biological},
inflammatory bowel disease\cite{jostins2012host} and 
Alzhiemer's disease\cite{lambert2013meta}.

A full list of studies, phenotypes and references is provided in supplementary table NNN.
As a robustness check, we estimated genetic correlation using both naive LD Score regression
and a more sophisticated model that allows for both MAF- and LD-dependent genetic architectures.
As expected from our simulation results, results from both models are generally consistent (supplementary figure NNN). 
However, the simpler model gives much lower standard error when applied to real data, especially for smaller GWAS, 
and performed better in simulations as measured by MSE,
we report results from the naive model hereafter (results from the more sophisticated model are displayed in Supplementary Figure NNN).

Note that LD Score regression heritability estimates are biased downwards by genomic control correction,
so we cannot report heritability estimates for the phenotypes in our dataset. 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Discussion}\label{Discussion}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\

Recap of the highlights

Main point: it is now almost trivial (mod admixed or non european GWAS)
to produce the all phenotype by all phenotype matrix of genetic correlations without 
the ethical issues around sharing genotypes IF people are willing to share INFO
(or at least provide a file with QC+ INFO > 0.9 SNPs)

\newpage
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Online Methods}\label{Online Methods}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Statistical Framework}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

See the supplementary note for a thorough exposition of the models behind LD Score regression.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Estimation of LD Scores}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

We estimated LD Scores from the European samples in the 1000 Genomes Project \cite{10002012integrated}
reference panel using the {-}{-}l2 flag in the ldsc software package by the authors (URLs) as in \cite{buliksullivan2014}.
We estimated per-allele LD Scores using the {-}{-}per-allele flag in ldsc, and
we estimated MAF-binned LD Scores using the {-}{-}cts-bin and {-}{-}cts-breaks flags in ldsc.
Following \cite{buliksullivan2014}, we estimated LD Scores using a 1 centiMorgan (cM) window
(with the ldsc flag {-}{-}ld-wind-cm 1).
Unlike \cite{buliksullivan2014}, we used a MAF cutoff of $1\%$ when estimating LD Scores,
in order to reduce the impact of LD measurement error on our regressions.
Since we only include variants with MAF $> 5\%$ in LD Score regressions for estimating genetic correlation,
and because there is very little LD between variants with MAF $> 5\%$ and variants with MAF $< 5\%$, 
this is unlikely to impact our results. For the analyses with HapMap 3 \cite{international2010integrating} LD Scores,
we took the sum of $r^2$'s over the same subset of HapMap 3 SNPs retained for LD Score regression in
\cite{buliksullivan2014}, (that is, HapMap 3 SNPs with MAF $> 1\%$, excluding centromeres and regions with long-range LD)
using the {-}{-}keep flag in ldsc.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Quality Control}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Imputation error can bias LD Score regression estimates of heritability and genetic covariance (Supplementary Note), though the biases in the numerator and denominator of the genetic correlation estimates will tend to cancel, so genetic correlation estimates are more robust to imputation error (Supplementary Figure NNN). To minimize this bias, we restricted to SNPs with reported INFO $> 0.9$ in all analyses, and we recommend restricting to INFO $>0.9$ as best practice for all LD Score regressions.

Genomic control correction biases LD Score regression estimates of heritability and genetic covariance downwards
(see the Supplementary Note of \cite{buliksullivan2014});
however, the bias in the numerator and denominator of the genetic correlation estimates will cancel, 
so genomic control correction will not bias LD Score regression estimates of genetic correlation.
Nevertheless, we wished to obtain accurate heritability estimates as well as genetic correlation estimates,
so we undid meta-analysis level genomic control correction by re-inflating all test statistics by multiplying by the reported 
$\lambda_{GC}$ correction factor.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{LD Score Regression}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection{Regression Weights} 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

For heritability estimation, we use the LD Score regression weights derived in the 
supplementary note from \cite{buliksullivan2014}. 
The optimal regression weights for genetic covariance estimation are 
$$ \var[\bhat_j\chat_j \,|\, \ell_j ] = 
	\left( 
		\frac{\hsqo\ell_j}{M} 
		+ 
		\frac{1-\hsqo}{N_1} 
	\right) \left(  
		\frac{\hsqt\ell_j}{M} 
		+ 
		\frac{1-\hsqt}{N_2}
	\right) 
	+ 					
	2\left( 
		\frac{\gencov\ell_j}{M} 
		+ 
		\frac{\rho N_s}{N_1N_2} 
	\right);
$$
(Supplementary Note) however, this quantity depends on both heritabilities, 
the genetic covariance and the number of overlapping samples,
which are often not known a priori, so some approximation is required.
In order to obtain approximate regression weights, 
we use heritability estimates from the single-phenotype LD Score regressions, then
we assume that $N_s$ is close enough to zero that the term $\rho N_s/N_1N_2$ is negligible
(though this default can be adjusted using the {-}{-}overlap and {-}{-}rho flags in ldsc),
and estimate a rough genetic covariance (which we only use for the regression weights)
using the aggregate estimator 
$$\hat{\rho}_{g, agg} := \frac{1}{\lbar}\sum_{j=1}^M \bhat_j\chat_j,$$
where $\lbar$ denotes the mean LD Score among SNPs included in the regression.
These regression weights are only an approximation to the optimal weights,
but this will not introduce bias into the regression;
it will only increase the standard error. 
The standard errors for LD Score regressions with summary statistics 
from GWAS with $N > 10,000$ are low enough to be interpretable,
so non-optimality of the regression weights does not seem to be a major hindrance.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection{Genetic Correlation}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Genetic correlation is defined as a ratio of quantities: 
$$r_g := \frac{\gencov}{\sqrt{\hsqo\hsqt}}.$$
The naive estimator of this ratio, 
$$\rhat_{g,naive} := \frac{\hat{\rho}_g }{ \sqrt{\hat{h}^2_1\hat{h}^2_2}},$$
is biased, and it is difficult to estimate a standard error for this estimator. 
Therefore, we use the block jackknife estimator of the ratio,
which is less biased:
$$
\hat{r}_{g,jackknife} := 
n_b\hat{r}_{g,naive} -
 \frac{n_b-1}{n_b}
 \sum_{i=1}^{n_b}
 	\frac{\hat{\rho}_{g,i} }
		{ \sqrt{\hat{h}^2_{1,i}\hat{h}^2_{2,i}}},
$$
where $n_b$ is the number of blocks, and $\hat{\rho}_{g,i}, \hat{h}^2_{1,i}$ and $\hat{h}^2_{2,i}$
are the estimates of genetic covariance and heritability obtained by deleting the $i^{th}$ block.
Our standard error estimates are also obtained from the block jackknife:
$$
\widehat{se}\left[\hat{r}_{g,jackknife}\right] := 
\sqrt{n_b} \sum_{i=1}^{n_b} \left(
 	\frac{\hat{\rho}_{g,i} }
		{ \sqrt{\hat{h}^2_{1,i}\hat{h}^2_{2,i}}} - \mean_i\left[
			\frac{\hat{\rho}_{g,i} }
			{ \sqrt{\hat{h}^2_{1,i}\hat{h}^2_{2,i}}}\right]
	\right).
$$
Block jackknife standard errors are robust to the  correlated error structure 
of GWAS $\chi^2$-statistics, 
so long as the block size exceeds the typical range of LD
(see \cite{buliksullivan2014} and \cite{moorjani2011history} 
for examples of papers in the statistical and population genetics literature that use this technique). 
We checked the reliability of our standard errors via simulations with real genotypes
(Supplementary Table NNN),
and found that the ldsc default setting  of 2000 blocks genome-wide
(which can be adjusted with the {-}{-}num-blocks flag) gives standard error estimates
that agree well with the empirical standard deviation across simulation replicates.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{GWAS Data}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection{IGAP}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

International Genomics of Alzheimer's Project (IGAP) is a large two-stage study based upon genome-wide association studies (GWAS) on individuals of European ancestry. 
In stage 1, IGAP used genotyped and imputed data on 7,055,881 single nucleotide polymorphisms (SNPs) to meta-analyze four previously-published GWAS datasets consisting of 17,008 Alzheimer's disease cases and 37,154 controls 
(The European Alzheimer's disease Initiative, EADI; the Alzheimer Disease Genetics Consortium, ADGC; The Cohorts for Heart and Aging Research in Genomic Epidemiology consortium, CHARGE; The Genetic and Environmental Risk in AD consortium, GERAD). 
In stage 2, 11,632 SNPs were genotyped and tested for association in an independent set of 8,572 Alzheimer's disease cases and 11,312 controls. 
Finally, a meta-analysis was performed combining results from stages 1 and 2.
Note that we only used stage 1 data for LD Score regression.

\newpage
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\input{./tex/main/legends}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{URLs}\label{URLs}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{enumerate}
	\item \texttt{ldsc} software:\\ 
		\texttt{github.com/bulik/ldsc}
		
	\item LD block genotype simulation code:\\
		\texttt{github.com/bulik/ldsc-sim}
		
	\item PGC (psychiatric) summary statistics:\\ 
		\texttt{www.med.unc.edu/pgc/downloads}
		
	\item GIANT (anthopometric) summary statistics: \\
		\texttt{www.broadinstitute.org/collaboration/giant/index.php/GIANT\_consortium\_data\_files}
		
	\item MAGIC (insulin, glucose) summary statistics: \\
		\texttt{www.magicinvestigators.org/downloads/}
		
	\item CARDIoGRAM (coronary artery disease) summary statistics:\\	
		\texttt{www.cardiogramplusc4d.org}
	
	\item DIAGRAM (T2D) summary statistics:\\
		\texttt{www.diagram-consortium.org}
		
	\item Rheumatoid Arthritis summary statistics:\\
		\texttt{www.broadinstitute.org/ftp/pub/rheumatoid\_arthritis/Stahl\_etal\_2010NG/}
	
	\item IGAP (Alzheimers) summary statistics:\\
		\texttt{www.pasteur-lille.fr/en/recherche/u744/igap/igap\_download.php}

	\item IIBDGC (inflammatory bowel disease) summay statistics:\\
		\texttt{www.ibdgenetics.org/downloads.html}\\
		Note that we used a newer version of these data with 1000 Genomes imputation.

	\item Plasma Lipid summary statistics:\\
		\texttt{www.broadinstitute.org/mpg/pubs/lipids2010/}
	
	\item Coffee:\\
		\texttt{barismo.com}
\end{enumerate}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\input{./tex/main/acknowledgements}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\input{./tex/main/author_contributions}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\input{./tex/main/competing_financial_interests}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\newpage
\bibliographystyle{plain}
\bibliography{gencor}

\end{document}
